cmd/operator/main.go

```diff
@@ -91,8 +101,12 @@ func init() {
 	flag.BoolVar(&flCLI, "cli", false, "run as CLI application to manage rollout in intervals")
 	flag.DurationVar(&flCLILoopInterval, "cli-run-interval", 60*time.Second, "the time between each rollout process (in seconds)")
 	flag.StringVar(&flHTTPAddr, "http-addr", defaultAddr, "address where to listen to http requests (e.g. :8080)")
+	flag.StringVar(&flPlatform, "platform", config.PlatformManaged, "target platform to query for Knative services (managed or gke)")
 	flag.StringVar(&flProject, "project", "", "project in which the service is deployed")
 	flag.StringVar(&flLabelSelector, "label", "rollout-strategy=gradual", "filter services based on a label (e.g. team=backend)")
+	flag.StringVar(&flGKEClusterLocation, "cluster-location", "", "zone in which the cluster is located")
+	flag.StringVar(&flGKECluster, "cluster", "", "ID of the cluster")
+	flag.StringVar(&flGKENamespace, "namespace", "default", "Kubernetes namespace where to look for Knative services")
```

Comment:
if you have time, it would be good to implement "all namespaces" and not have this flag at all.

Reply:
I'll try this out. The way I structured #107 should make it feasible to make this happen